rpm_spec/subpackages/manageiq-core

```diff
@@ -18,6 +18,10 @@ Requires: socat
 %description core
 %{product_summary} Core
 
+%pre core
+# ensure this user exists (build and upgrade)
+/usr/bin/id manageiq > /dev/null 2>&1 || /usr/sbin/useradd manageiq && /usr/sbin/usermod -aG manageiq root
```

Comment:
Should we pick a UID & GID or at least specify `--system` with `useradd`?  Also, thoughts on home directory and login shell?

Reply:
system removes the home directory.

Not sure the implications of the uid / gid values